sortowanie kul.py

```diff
@@ -1,40 +1,51 @@
 #- wczytaj z pliku N lini z kolorami oddzielonymi przecinkami, wynikiem powinna być sekwencja ruchów, które pozwolą rozwiązać układ.
```

Comment:
also, try
pep8 or
pycodestyle
to test if your code matches formatting requirement.